### ISSUE TITLE
rocon_msgs depend on missing rocon-demo-msgs in Groovy

### ISSUE THREAD

**Issue by bit-pirate** [robotics-in-concert/rocon_msgs#21]
Looks like someone deleted those demo msgs, but forgot to remove them from the Groovy dependency list (they are removed in Hydro).

We need to re-release the messages for Groovy with that change.

**Comment by bit-pirate** [comment]
What about just releasing [0.5.1](https://github.com/robotics-in-concert/rocon_msgs/releases/tag/0.5.1) for Groovy? That version already has those demo messages removed (https://github.com/robotics-in-concert/rocon_msgs/commit/7ec4e50cec034ade8811b3bd2b2624fd5941410b).

**Comment by stonier** [comment]
Original email copy: https://groups.google.com/a/rnd.yujinrobot.com/forum/#!forum/rocon

```
Hi,

my name is Thomas Kuntz, and i work in a french research 
team (Network Team of ICube lab) as a student, and we
tried to install the ros-groovy-rocon package on our turtlebot,
but we alway run into a dependency problem : 
ros-groovy-rocon-msgs can't be installed. And when we
tried to look why, we found out that ros-groovy-rocon-msgs
depends on ros-groovy-rocon-demo-msgs, but
ros-groovy-rocon-demo-msgs isn't available, maybe you
removed it from the repository? If that so, then maybe
you forget to remove the dependency ? If not, do you
have any idea why we can't install it on our turtlebot?

Thank you for you time, have a nice day.
```

**Comment by stonier** [comment]
I re-released hydro at 0.6.0 and bugfixed groovy-devel at 0.5.3 so we can bugfix and bump groovy separately to hydro without awkward version number conflicts.
